server/core/src/main/scala/org.apache.daffodil.debugger.dap/DAPodil.scala

```diff
@@ -67,9 +65,9 @@ object DAPSession {
       def sendEvent(event: DebugEvent): IO[Unit] =
         Logger[IO].info(show"<E $event") *> IO.blocking(server.sendEvent(event))
 
-      def terminate(message: String): IO[Unit] =
-        sendEvent(new ErrorEvent(s"${message}. check debugger output for more information.")) *>
-          sendEvent(new Events.TerminatedEvent())
+      /** Send DebugEvent back to extension and exit session, ending debug */
+      def abort(event: DebugEvent): IO[Unit] =
```

Comment:
yeah that is correct. they all caused the program to stop. I will remove the message param from all ErrorEvents, that makes a look of sense. I will make case objects instead of case classes as well.